sites/fast-website/src/app/style/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const siteBreakpoints: string[] = [
```

Comment:
The way I've usually done it was to give the sizes some specific names that correspond to the type of device:

```ts
export const tinyDeviceWidth: number = 320;

export const midDeviceWidth: number = 700;
export const midDeviceWidthMax: number = midDeviceWidth - 1; // upper bound for devices smaller than 700px
export const midDeviceWidthMin: number = midDeviceWidth + 1; // lower bound for devices larger than 700px

export const normalDeviceWidth: number = 1000;
export const normalDeviceWidthMax: number = normalDeviceWidth - 1; // upper bound for devices smaller than 1000px
export const normalDeviceWidthMin: number = normalDeviceWidth + 1; // lower bound for devices larger than 1000px

export const bigDeviceWidth: number = 1200;

// we could add extra named variables as needed without worry:
// export const weirdDeviceWidth: number = 672;
// export const hugeDeviceWidth: 1600;
```

Reply:
This definitely makes sense and provides better readability.  What do you think of a siteBreakpoint object with each of these as properties vs individual exports and imports?

Reply:
I think we should discuss the best way forward. In react we have used an array to represent the app breakpoints using `BreakpointTracker`. I think the problem with naming these is that the names are kind of meaningless and can get messy pretty quickly. @freefaller69 maybe post up in Discord to gather opinions on how we want to track breakpoints.

Reply:
This has been hanging out for awhile and it seems like we don't have full resolution on this. I think @radium-v is going to write a proposal for handling breakpoints in our new reality of W/C's and the modern web and I'd prefer to resolve this in that issue. For now, let's move ahead with *something* for this specific implementation so we aren't blocked on the ideal path forward for FAST in the future. We can add a TODO that links to the issue and that will help us follow-up once we land the "blessed" thing.

My primary concern here is that we land the responsiveness of this so we can move ahead with the site and not wait to land our overall principle for how we handle media queries overall as that convo could take a bit to resolve.

Reply:
Reverting this to fixed numbers for for the sake of readability with John's forthcoming proposal on a uniform reusable approach.